lib/airbrake/logger_backend.ex

```diff
@@ -25,6 +25,7 @@ defmodule Airbrake.LoggerBackend do
         context: err_info[:context]
       )
     rescue
+      # credo:disable-for-next-line
       exception -> IO.inspect(exception)
```

Comment:
credo doesn't like `IO.inspect/1`, but I don't know what else to replace it with here.  We're already in a logger!